app/modules/individuals/schemas.py

```diff
@@ -41,6 +41,26 @@ class Meta:
         dump_only = (Individual.guid.key,)
 
 
+class CreateIndividualSchema(BaseIndividualSchema):
+    """
+    Create Individual schema for just the fields needed at creation
+    """
+
+    # sex = base_fields.Function(lambda ind: ind.get_sex())
+
+    class Meta(BaseIndividualSchema.Meta):
+        fields = BaseIndividualSchema.Meta.fields + (
+            Individual.created.key,
+            Individual.updated.key,
+            Individual.sex.key,
```

Comment:
This works but can someone @bluemellophone please explain why the other option, 'sex' and the line above (which was taken from the DetailedIndividualSchema meant that if (like the FE does) 'null' is passed, which is Python None, this fails with 
`{
    "status": 422,
    "messages": {
        "sex": [
            "Field may not be null."
        ]
    },
    "message": "The request was formatted correctly but contains a semantic error (check input values and types)."
}
`

Reply:
i do not feel i have enough understanding of the depths of the flask/patch internals to be able to answer this.

Reply:
The reason why `DetailedIndividualSchema` doesn't work is because `sex`
is defined like this:

```
sex = base_fields.Function(lambda ind: ind.get_sex())
```

The field looks like this:

```
<fields.Function(default=<marshmallow.missing>, attribute=None, validate=None, required=False, load_only=False, dump_only=False, missing=<marshmallow.missing>, allow_none=False, error_messages={'required': 'Missing data for required field.', 'type': 'Invalid type.', 'null': 'Field may not be null.', 'validator_failed': 'Invalid value.'})>
```

if we remove this definition, the field looks like this:

```
<fields.String(default=<marshmallow.missing>, attribute=None, validate=[<Length(min=None, max=40, equal=None, error=None)>], required=False, load_only=False, dump_only=False, missing=<marshmallow.missing>, allow_none=True, error_messages={'required': 'Missing data for required field.', 'type': 'Invalid type.', 'null': 'Field may not be null.', 'validator_failed': 'Invalid value.', 'invalid': 'Not a valid string.', 'invalid_utf8': 'Not a valid utf-8 string.'})>
```

The important difference is `allow_none=False` vs `allow_none=True`.

Reply:
Thankyou and well done on finding that

Reply:
wow yeah that is some sleuthing.  :tada:

Reply:
By the way it works if we add `allow_none=True`

```python
sex = base_fields.Function(lambda ind: ind.get_sex(), allow_none=True)
```